session8/src/com/itschool/homework/EX6.java

```diff
@@ -0,0 +1,11 @@
+package com.itschool.homework;
+
+public class EX6 {
+
+    public static void main(String[] args) {
+
+        StringBuilder stringBuilder = new StringBuilder("The sun is up");
+        stringBuilder.delete(4, 7);
+        System.out.println("Updated string: " + stringBuilder.toString());
```

Comment:
I am not sure what this program is supposed to do, because as far as I see it just deletes something from the text. But the exercise sais:
_Use the StringBuilder class to replace every occurrence of a substring within a given string with another substring. For instance, replace "cat" with "dog" in the string "The cat sat on the mat."_
Could you explain?